client/related_items/helpers.js

```diff
@@ -28,14 +28,12 @@ const findDefaultStyle = (styles) => {
   return new Promise (resolve => {
```

Comment:
Just a thought--You don't need to create Promises out of this synchronous helper function. You could make this sync and stick it inside of a .then block to cut down on some of your promise chaining.

Reply:
Ah really good catch! Will work on that for the next PR 👍